From 77b494c6371063aa7e92e6f59baf735eeba22c60 Mon Sep 17 00:00:00 2001
From: Moritz Fischer <mdf@kernel.org>
Date: Tue, 24 Jul 2018 16:01:40 -0700
Subject: [PATCH 17/17] mfd: cros_ec: Fix struct definition to work with
 upstream

Fix struct definition with missing parts in order to make
thermal commands work.

Fixes 5271db29d71 ("mfd: cros_ec: Sync to the latest cros_ec_comm ...")
Signed-off-by: Moritz Fischer <mdf@kernel.org>
---
 include/linux/mfd/cros_ec_commands.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/linux/mfd/cros_ec_commands.h b/include/linux/mfd/cros_ec_commands.h
index e40d66d09aa9..ec127c6f8d42 100644
--- a/include/linux/mfd/cros_ec_commands.h
+++ b/include/linux/mfd/cros_ec_commands.h
@@ -1810,6 +1810,7 @@ enum ec_temp_thresholds {
  */
 struct ec_thermal_config {
 	uint32_t temp_host[EC_TEMP_THRESH_COUNT]; /* levels of hotness */
+	uint32_t temp_host_release[EC_TEMP_THRESH_COUNT]; /* release levels */
 	uint32_t temp_fan_off;		/* no active cooling needed */
 	uint32_t temp_fan_max;		/* max active cooling needed */
 } __packed;
-- 
2.20.1

